src/core/Expander/ExpanderGroup/ExpanderGroup.baseStyles.tsx

```diff
@@ -22,9 +22,6 @@ export const baseStyles = (theme: SuomifiTheme) => css`
       transition: margin ${`${theme.transitions.basicTime}
         ${theme.transitions.basicTimingFunction}`};
 
-      & .fi-icon {
```

Comment:
Since after the removal the correct styling is passed through the color prop of the parent element (ExpanderTitleButton), would it be more robust to explicitly add the correct styling via the `fi-icon` class name? Probably not gonna make a big difference in practice, but wanted to put the thought out there.

Reply:
As discussed elsewhere, the proposed change would not provide much additional value, so we can leave this as it is with the current changes.